generator/internal/tracing_connection_generator.cc

```diff
@@ -249,6 +251,11 @@ std::unique_ptr<::google::cloud::AsyncStreamingReadWriteRpc<
     return R"""(
 StreamRange<$response_type$>
 $tracing_connection_class_name$::$method_name$($request_type$ const& request) {
+  auto span = internal::MakeSpan("$product_namespace$::$connection_class_name$::$method_name$");
+  auto scope = opentelemetry::trace::Scope(span);
+  auto sr = child_->$method_name$(std::move(request));
+  return internal::MakeTracedStreamRange<$response_type$>(
+        std::move(span), std::move(sr));
   return child_->$method_name$(request);
```

Comment:
???

Reply:
Redundancy is a fundamental concept in cloud computing.

.... Fixed. Also realized that this method accepts the request by `const&`, whereas the paginated API accepts the request by value. So I stopped moving it.